src/w3c/headers.js

```diff
@@ -663,17 +663,15 @@ export function run(conf, doc, cb) {
       $("<meta lang='' property='dc:language' content='en'>")
     );
   }
-  // insert into document and mark with microformat
-  var bp;
-  if (conf.isCGBG) bp = cgbgHeadersTmpl(conf);
-  else bp = headersTmpl(conf);
-  $("body", doc)
-    .prepend($(bp))
-    .addClass("h-entry");
+  // insert into document
+  const bp = hyperHTML.bind(document.createDocumentFragment())`${
+    [(conf.isCGBG ? cgbgHeadersTmpl : headersTmpl)(conf)]}`;
```

Comment:
I think you may want a `.wire(obj)` here. On my phone reviewing, so kindly please check the hyperHTML docs.

Reply:
`.wire(obj)` here returns an hyperHTML-specific object that contains a `_hyper` comment. I really don't like that behavior, but anyway this is a workaround to get a single node.

Otherwise we can do this:

```js
const bp = hyperHTML`${...}`; // hyperHTML can be used as an alias for hyperHTML.wire()
document.body.insertBefore(bp.childNodes[0], document.body.firstChild);
```

This may break if we choose to insert multiple top elements to the header template by whatever reason, but still works.

Reply:
Can you help me understand your concerns about the comment nodes? Are you worried they will end up in the final spec?

Reply:
>Are you worried they will end up in the final spec?

Partially yes, as my experiment shows a lot of resulting comments (one for every if-branch) when porting the full header templates.

But in this specific PR, I just don't like the result of ``hyperHTML`${[...]}` `` is not an HTMLElement just because of that additional comment node.